.gitignore

```diff
@@ -29,3 +29,4 @@ mediaroot/*
 node_modules
 parts
 staticroot/*
+bower_components
```

Comment:
Since we want to match it only in the project root, it should be: `/bower_components/`